resources/views/livewire/auth-navigation-menu.blade.php

```diff
@@ -43,7 +43,7 @@ class="bg-white dark:bg-gray-900 border-b border-gray-100 dark:border-gray-700 r
 
```

Comment:
Seems like I'm missing the "Contact" navigation link when I compare this with the responsive menu. Suggestion: turn this into a Blade component and specify the menu links in some array. Then you can loop over the array for both menus